src/app/public/src/modules/open-on-click-directive/open-on-click.directive.ts

```diff
@@ -0,0 +1,26 @@
+import { Directive, HostListener, Input, Renderer2, ElementRef } from '@angular/core';
+
+import { HelpWidgetService } from '../shared';
+
+@Directive({
+  selector: '[bbHelpOpenOnClick]'
+})
+export class BBHelpOpenOnClickDirective {
+  @Input()
+  private bbHelpOpenOnClick: string;
+
+  constructor (
+    private widgetService: HelpWidgetService,
+    private elementRef: ElementRef,
+    private renderer: Renderer2) {
+      this.renderer.setStyle(this.elementRef.nativeElement, 'cursor', 'pointer');
+  }
+
+  @HostListener('click', ['$event'])
+  public onClick(event: MouseEvent): boolean {
```

Comment:
If this directive is placed on a non-button or non-anchor element, it will not work with the keyboard. You could probably add a `keydown` event for that bit...

Reply:
Good thought! I added the event listener, and looked for `enter` or `space`;  are there other keys I should be looking for?